### PR TITLE
Add button to clear all update requests

### PR DESCRIPTION
**Pull request by calumats**
merge after PR #319

### CODE REVIEW

**Comment by calumats** [comment]
retest this

File: web-server/app/assets/javascripts/components/updates/list-of-updates.jsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ define(function(require) {
       Router = require('react-router'),
       db = require('../../stores/db'),
       React = require('react');
+      Fluxbone = require('../../mixins/fluxbone'),
+      sendRequest = require('../../mixins/send-request'),
```
Comment (jerrytrieu) [inline web-server/app/assets/javascripts/components/updates/list-of-updates.jsx]: semi-colons?

**Comment by calumats** [comment]
Fixed, good catch.

**Comment by calumats** [comment]
TC seems to not want to run tests on this branch however.

**Pull request merged**